packages/app-content-pages/src/screens/About/About.js

```diff
@@ -1,18 +1,17 @@
 import { Box } from 'grommet'
 import { useTranslation } from 'next-i18next'
-import styled from 'styled-components'
 import Script from 'next/script'
 import { useState } from 'react'
 
-import PageLayout from '@shared/components/PageLayout/layout.js'
-import DropdownNav from '@shared/components/DropdownNav/DropdownNav.js'
-import Head from '@shared/components/Head'
-import Sidebar from '@shared/components/Sidebar/Sidebar.js'
-import MaxWidthContent from '@shared/components/MaxWidthContent/MaxWidthContent.js'
+import PageLayout from '../../shared/components/PageLayout/layout.js'
```

Comment:
For my own edification.. Why are we removing the `@` for the import?

Reply:
Similar to something Mark ran into with lib-user, I was having trouble with the `@shared` path being resolved in components built through storybook and then used in unit tests. I decided to just standardize everything to use the relative path.